tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = flake8, mypy, py38, py37, py36, docs
```

Comment:
@cyenyxe please also update the readme so users run `tox -e py3.*` rather than `tox -e py3*`